app/models/chat.rb

```diff
@@ -0,0 +1,4 @@
+class Chat < ApplicationRecord
+  belongs_to :match
```

Comment:
You should validate the presence of the `match` association.
Also, given there is a `has_one :chat` on `Match`, you should validate the uniqueness of the `match_id`